docs/src/howto-guides/020_monitor_tedge_health.md

```diff
@@ -7,16 +7,37 @@ The health endpoints conform to the following topic scheme, listening for health
 
 `tedge/health-check/<tedge-daemon-name>`
 
-expecting empty messages, triggering the health check.
-
-The daemon will then respond back on the topic:
+expecting empty messages, triggering the health check.The daemon will then respond back on the topic:
```

Comment:
```suggestion
expecting empty messages, triggering the health check. The daemon will then respond back on the topic:
```

Reply:
Ignoring my suggestions as we had an interactive session to reformat larger sections of the document which covers this.

Reply:
Ignoring my suggestions as we had an interactive session to reformat larger sections of the document which covers this.